torch/_dynamo/variables/torch.py

```diff
@@ -243,6 +243,12 @@ def call_function(
             ).call_function(tx, args, kwargs)
         elif self.value in config.constant_functions:
             assert not args and not kwargs
+            # See: https://github.com/pytorch/pytorch/issues/110765
+            if self.value in [
```

Comment:
@voznesenskym not clear if this is sufficient. Should we just mark side effects for all `config.constant_functions`?

Reply:
Not sure what else we might want to make as `inconsistent_side_effects`

Reply:
Anw, we can leave this as a future step, fixing the `is_compiling` case is a good step forward at least.

Reply:
Your other option is to actually run the function and then compare it against what was requested. This requires (1) that the Dynamo compilation context actually mimics the ambient context exactly, or you have some way of restoring to the ambient context (I think, in practice, all of the currently present constant functions would be safe to run this way) and (2) the constant function has to be actually safe to run.

Maybe just doing all constant functions is a better idea...

Reply:
@voznesenskym would you mind taking a look at this?

Reply:
All constant functions is fine, this is also fine as is.